lib/data/landing_page_content_items/goals.yaml

```diff
@@ -103,7 +100,6 @@ blocks:
         <h2>Dorem sit</h2>
         
         <p>TODO - social media links</p>
```

Comment:
This should be removed as well as `share_links` block has already been added.

Reply:
Ah damnit, I saw that and then got distracted by the dog 🤦🏻  - sorry!

Reply:
Fixed in [this force push](https://github.com/alphagov/frontend/compare/1815f563ce1d668af074f9f2f68b3a4f0dfef4b4..81b239c3caf5de6cd31b8bc0f2a9c5c55e8373a5)